METdbLoad/ush/read_data_files.py

```diff
@@ -630,6 +630,23 @@ def read_data(self, load_flags, load_files, line_types):
                                          (all_stat['8'].isnull()) &
                                          (~all_stat['5'].isnull()), '5']
 
+                # Some lines in stat files may be missing ec_value
+                # CTC and CTS, set to .5
+                # MCTS, set to 1/n_cat. MCTC is variable length
+                if all_stat[CN.LINE_TYPE].eq(CN.CTC).any():
+                    all_stat.loc[(all_stat.line_type == CN.CTC) &
+                                 (all_stat['5'].isnull()), '5'] = .5
+
+                if all_stat[CN.LINE_TYPE].eq(CN.CTS).any():
+                    all_stat.loc[(all_stat.line_type == CN.CTS) &
+                                 (all_stat['96'].isnull()), '96'] = .5
```

Comment:
See comment above

Reply:
I checked the Python docs to see if both isna and isnull are needed in pandas. The definitions are the same, and an entry in Stack Overflow says that they are literally the same code. They both check for null or missing values.